components/metal/IntroSection.vue

```diff
@@ -54,6 +64,7 @@ export default class extends Vue {
 
   &__title {
     @include type-style('expressive-heading-05');
+    max-width: 32rem;
```

Comment:
For `max-width` property we are using the columns size variable for each screen. Look the example on line 87: `max-width: 6 * $column-size-large;`. Could you change this `32rem` with the proper value using `$column-size-large;`?